lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -3740,7 +3780,8 @@ impl ClientDriveQueryDirectoryResponse {
                 FileInformationClass::FileFullDirectoryInformation(fs_info_class) => {
                     fs_info_class.size()
                 }
-                // TODO(isaiah): add support for FileDirectoryInformation and FileNamesInformation
+                FileInformationClass::FileNamesInformation(fs_info_class) => fs_info_class.size(),
+                // TODO(isaiah): add support for FileDirectoryInformation
```

Comment:
What's the impact of not doing this now? Is there some other case that will fail?

Reply:
It's unknown -- I haven't encountered a case that requires it during testing, but FreeRDP implements it. It's simple to add, so I'll just tack it on here for completeness.

Reply:
Added in 49e0cf33458832855eafd4516bce5ea729d95110